### PR TITLE
Mobile version portfolio

### PR DESCRIPTION
**Pull request by Surafels**
The project shows A portfolio template for a mobile version.

  changes:
- Llinters added.
- Index.html  
- styles.css.
- fix linters errors.
- Readme updated.

### CODE REVIEW

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @Surafels, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive pull request and a README.md file that doesn't meet the project requirements. Kindly, I suggest fixing these issues before asking for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by zeff96** [comment]
Hi @Surafels, It seems you are struggling to get the concept, If you need help kindly reply here so we can connect ASAP

File: README.md [hunk 1]
```diff
@@ -27,37 +27,85 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (zeff96) [inline README.md:110]: - [ ] This link is broken, Kindly fix it before requesting another review. From the files in your directory, I can see the license file named `MIT.md` Kindly rectify this link to point to mit not `LICENSE` which is not present in your directory

**Review by zeff96 — COMMENTED** [review]
Hi @Surafels , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a closed PR, I would advise you to consider reopening the PR and ask for another review. Additionally, your `MIT` link is broken. Kindly fix the same before asking for another review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**